source/includes/language-compatibility-table-cxx.rst

```diff
@@ -10,6 +10,13 @@
      - C++11
      - C++03
 
+   * - 3.9.0
```

Comment:
S: instead of adding a new entry, what if you changed the previous entry to be `>3.8`

Reply:
love it